docs/packages.md

```diff
@@ -0,0 +1,101 @@
+# LinuxKit packages
```

Comment:
I'll add a bunch more documentation about packages. This is just the basics so that folks know how to build